### ISSUE TITLE
Wrong render?

### ISSUE THREAD

**Issue by katopz** [developit/preact-compat#78]
I'm trying apply `preact-compat` to [redux-undo example](https://github.com/katopz/redux-undo/tree/preact-compat) by `alias` which seem to working fine. only on thing that's not. Let's get in to the code.

Here's working via `React` but not `Preact compat`
```js
const Link = ({ active, children, onClick }) => {
  if (active) {
    return <span>{children}</span>
  }

  return (
    <a href="#"
       onClick={e => {
         e.preventDefault()
         onClick()
       }}
    >
      {children}
    </a>
  )
}
```
It'll appear blank (not render?) when click on category link like this

![screen shot 2016-06-20 at 21 08 33](https://cloud.githubusercontent.com/assets/97060/16197624/fca7ad44-372c-11e6-8d97-0c3371c54a7e.png)

Here's working both
```js
const Link = ({ active, children, onClick }) => {
  if (active) {
    return <span>{children}</span>
  }

  return (<span>
    <a href="#"
       onClick={e => {
         e.preventDefault()
         onClick()
       }}
    >
      {children}
    </a>
  </span>)
}
```

The different is I've to add `return (<span>...</span>)` to make it render properly for some reason.
Any hint? Or maybe just live with it?

Thanks

**Comment by developit** [comment]
@katopz What version of preact is this happening with?

**Comment by katopz** [comment]
@developit It's [appear](https://github.com/katopz/redux-undo/blob/preact-compat/examples/todos-with-undo/package.json#L18) to be
```
    "preact": "^4.8.0",
    "preact-compat": "^1.7.1",
```
I try update but still had this same issue.
```
    "preact": "^4.8.0",
    "preact-compat": "^1.8.0",
```

Thanks

**Comment by developit** [comment]
Can you try installing the beta? Current is 5.0.1-beta.14

**Comment by developit** [comment]
`npm install --save preact@beta` should work. Npm 2 might give you a warning but ignore it.

**Comment by katopz** [comment]
I'm on `beta` now, it seem worst because it appear only `redo` button now.

**Comment by developit** [comment]
@katopz [from gitter]: Just published `5.0.1-beta.15` that I'm hoping will fix the issue you noted when updating to the beta. LMK :)

**Comment by developit** [comment]
Confirmed with @katopz that this has been fixed (including the lovely #204 discovery) as of `5.1.0-beta.16`.
